classport-commons/README.md

```diff
@@ -0,0 +1,10 @@
+# classport-commons
+
+This module contains the classes for reading the annotation embedded into the class files and for building the output of classport, i.e., the dependencies list or the depependencies tree.
+
+In detail:
+* `AnnotationReader.java` is about parsing the annotations
+* `ClassportHelper.java` is for getting ClassportInfo's fields
+* `ClassportInfo.java` contains the annotation
+* `ClassportProject.java` is for building the final output of Classport tool
```

Comment:
Good that you document this but you do not need to explain all files here. It is good for our understanding what this means.
You do not need to delete this since you have already written, but we also don't need to add more information here.